src/components/DropzoneAreaBase.js

```diff
@@ -369,10 +355,10 @@ export const FileObjectShape = PropTypes.shape({
 
 DropzoneAreaBase.propTypes = {
     /** @ignore */
-    classes: PropTypes.object.isRequired,
```

Comment:
I presume this was formerly required since we were using HOCs and so it actually became a prop, however now classes is obtained through the hook and so it makes no sense that this is required? Perhaps we can remove it altogether, I can't see how this would be used anymore? Any way just getting a second opinion on this to be safe

Reply:
Yes, you are totally right on this one, my bad.

Can be removed, also remove the `/** @ignore */` above 😉

Reply:
Ok, this change has been pushed